src/main/augurNodeController.js

```diff
@@ -33,8 +34,12 @@ const defaultConfig = {
 }
 
 function AugurNodeController() {
-    this.configPath = path.join(__dirname, '../config.json')
-    this.augurDbPath = path.join(__dirname, '../augur.db')
+    let appDataPath = appData("augur");
+    if (!fs.existsSync(appDataPath)){
```

Comment:
`fs.exists` and `fs.existsSync` are deprecated (see https://nodejs.org/api/fs.html#fs_fs_exists_path_callback), prob should use `fs.stat` instead.

Reply:
Apparently, `fs.existsSync` is _not_ deprecated.  Just `fs.exists`.  So all is well here!